pandas/_libs/parsers.pyx

```diff
@@ -655,8 +655,7 @@ cdef class TextReader:
                     raise ValueError('Multiple files found in compressed '
                                      'zip file %s', str(zip_names))
             elif self.compression == 'xz':
-                lzma = compat.import_lzma()
-
+                import lzma
```

Comment:
can we import at the top

Reply:
Sure, left the `import` there since all the other branches of the `if` statement had imports specific to the branch within them, so I assumed it was done that way for a reason.  Will move all the imports; can revert and just move this one if the other should remain where they are.